build.gradle

```diff
@@ -794,7 +795,14 @@ subprojects {
     description = 'Run checkstyle on all test Java sources'
   }
 
-  test.dependsOn('checkstyleMain', 'checkstyleTest')
+  // All available signatures https://github.com/policeman-tools/forbidden-apis/wiki/BundledSignatures
+  apply plugin: 'de.thetaphi.forbiddenapis'
+  forbiddenApis {
+    bundledSignatures = ['jdk-unsafe']
+    ignoreFailures = true
```

Comment:
@gharris1727  without this ignoreFailures extension, check fails for almost any module I see.